### PR TITLE
test: the basic structure should has stable output

### PR DESCRIPTION
**Pull request by bvanjoi**
## Summary

copilot:summary

## Details

copilot:walkthrough

## Checklist

- [ ] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: e2e/tests/basic.test.ts-snapshots/basic-test-should-has-stable-output-1-darwin.txt [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<h1 id="h1" class="text-3xl mb-10 leading-10 tracking-tight title_c6d79">h1<a class="link_c6d79 header-anchor" aria-hidden="true" href="#h1">#</a></h1>
+<h2 id="h2" class="mt-14 mb-6 text-2xl tracking-tight title_c6d79">H2<a class="link_c6d79 header-anchor" aria-hidden="true" href="#h2">#</a></h2>
+<h3 id="h3" class="mt-10 mb-2 leading-7 text-xl title_c6d79">H3<a class="link_c6d79 header-anchor" aria-hidden="true" href="#h3">#</a></h3>
```
Comment (bvanjoi) [inline e2e/tests/basic.test.ts-snapshots/basic-test-should-has-stable-output-1-darwin.txt:3]: why the `hash` in <a class="link_{hash}"/> is not stable?

Comment (sanyuan0704) [reply e2e/tests/basic.test.ts-snapshots/basic-test-should-has-stable-output-1-darwin.txt:3]: The hash generated by default behavior in rspack may be related to the filepath, which is different between local and ci environment. You can delete the class part in tags and then generate snopshot.

Comment (chenjiahan) [reply e2e/tests/basic.test.ts-snapshots/basic-test-should-has-stable-output-1-darwin.txt:3]: we can create an issue for Rspack

Comment (bvanjoi) [reply e2e/tests/basic.test.ts-snapshots/basic-test-should-has-stable-output-1-darwin.txt:3]: is it generated by css module?

Comment (sanyuan0704) [reply e2e/tests/basic.test.ts-snapshots/basic-test-should-has-stable-output-1-darwin.txt:3]: yes, the classname including hash is processd by css modules.

**Review by sanyuan0704 — APPROVED** [review]

**Pull request merged**